src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup.HydraAdapter/ClientProxyBase.cs

```diff
@@ -37,6 +38,8 @@ public class ClientProxyBase
         private CancellationTokenSource cancellationTokenSource = new CancellationTokenSource();
         public CancellationToken CmdletCancellationToken;
 
+        public static AzureRmRecoveryServicesVaultCreds recoveryServicesVaultCreds = new AzureRmRecoveryServicesVaultCreds();
```

Comment:
Maybe define a property with public get and private set to access this field recoveryServicesVaultCreds? The property maybe named like: VaultContext. I'm assuming this will be called by the Adapter methods? Can you cover the caller in a later PR, if not this PR?